### ISSUE TITLE
[T:Bug-Crash] App crashes only when remote debugging is enabled. 

### ISSUE THREAD

**Issue by moisesynfam** [realm/realm-js#2389]
My app crashes when it tries to open a realm while remote debugging is on.

## Goals

Run app and create a realm while remote debugging is on.

## Expected Results

App to run normally.

## Actual Results
I get this error on the Chrome console:
```
Error DOMException: Failed to execute 'send' on 'XMLHttpRequest': Failed to load 'http://192.168.1.102:8083/create_realm'.
    at makeRequest (blob:http://localhost:8081/d5d73e44-b03f-40ed-9fe8-a3471cc6bd04:108025:16)
    at sendRequest (blob:http://localhost:8081/d5d73e44-b03f-40ed-9fe8-a3471cc6bd04:108070:22)
    at Object.createRealm (blob:http://localhost:8081/d5d73e44-b03f-40ed-9fe8-a3471cc6bd04:107804:12)
    at new Realm (blob:http://localhost:8081/d5d73e44-b03f-40ed-9fe8-a3471cc6bd04:106408:25)
    at Function.open (blob:http://localhost:8081/d5d73e44-b03f-40ed-9fe8-a3471cc6bd04:106629:42)
    at _callee$ (blob:http://localhost:8081/d5d73e44-b03f-40ed-9fe8-a3471cc6bd04:104660:64)
    at tryCatch (blob:http://localhost:8081/d5d73e44-b03f-40ed-9fe8-a3471cc6bd04:41529:19)
    at Generator.invoke [as _invoke] (blob:http://localhost:8081/d5d73e44-b03f-40ed-9fe8-a3471cc6bd04:41704:24)
    at Generator.prototype.<computed> [as next] (blob:http://localhost:8081/d5d73e44-b03f-40ed-9fe8-a3471cc6bd04:41572:23)
    at tryCatch (blob:http://localhost:8081/d5d73e44-b03f-40ed-9fe8-a3471cc6bd04:41529:19)
```
## Steps to Reproduce

Enable remote debugger.

## Version of Realm and Tooling

- Realm JS SDK Version: 2.28.0
- React Native: 0.59.4
- Client OS & Version: Windows 10 / Android 9
- Which debugger for React Native: Chrome Remote debugger

**Comment by dekameron22** [comment]
+1

**Comment by max-zu** [comment]
The same in prod build

**Comment by LuigiMaestrelli** [comment]
+1

**Comment by paschaldev** [comment]
+1

**Comment by paschaldev** [comment]
For anybody facing this issue, this was how I solve the problem.

1. Install this version: ^2.28.1
2. Unlink realm: `react-native unlink realm`
3 Link realm again: `react-native link realm`

Problem solved

**Comment by HengCC** [comment]
is there any other solution? This problem occured when the remote debug mode is on. 

realm version:  2.28.1
react: 16.6.3,
react-native: 0.58.6,
node version:10.10

**Comment by HengCC** [comment]
@moisesynfam  Have you solved this problem?

**Comment by HengCC** [comment]
finnaly , i downgrade realm's  version to 2.26.1, now it's working fine.

**Comment by max-zu** [comment]
Issue was fixed in [2.29.0](https://github.com/realm/realm-js/releases/tag/v2.29.0) release.

**Comment by moisesynfam** [comment]
yes I can confirm it was fixed on 2.29.0
